rust/migration/src/m20250900_000000_add_content_hash_to_blob.rs

```diff
@@ -0,0 +1,40 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(Blob::Table)
+                    .add_column(
+                        ColumnDef::new(Blob::ContentHash)
+                            .string()
+                            .not_null()
+                            .default("")
```

Comment:
I always greatly prefer `None`/`NULL` as a fallback for strings rather than `""`.  Though the client requirement does make `NULL` a bit more of a hassle here.

Reply:
> we want it to be a hard requirement for blobs to have hashes for this verification to work

Let's not provide a default value here, then?  Prevent the inconsistent state from being specified to begin with, and all.